neuro-sdk/src/neuro_sdk/buckets.py

```diff
@@ -618,13 +618,15 @@ async def _get_bucket_fs(
         async with self._get_provider(bucket_name, cluster_name) as provider:
             yield BucketFS(provider)
 
-    def _split_blob_uri(self, uri: URL) -> Tuple[str, str, str]:
+    def split_blob_uri(self, uri: URL) -> Tuple[str, str, str]:
```

Comment:
Should the method be moved into `client.parse` namespace along with other URI helpers.